### DISCUSSION TITLE
How to add drawables?

### DISCUSSION THREAD

**Discussion by zt64** [REAndroid/ARSCLib#53] [Q&A]
How would I add drawables to the APK? Specifically ones in a png format to the res/drawables directory

**Comment by REAndroid** [comment]
it'd be easy to answer based on what you have tried. Check java example on [README](https://github.com/REAndroid/ARSCLib#examples) and this issue :https://github.com/REAndroid/ARSCLib/issues/44#issuecomment-1585699331
